packages/react-app/src/components/Modals/Wrap.jsx

```diff
@@ -4,12 +4,13 @@ import { useCallback, useState } from "react";
 import { FieldElement } from "./Add";
 import { ModalHeader, Row } from "./index.styled";
 import { useContractLoader } from "eth-hooks";
+import { toDecimals } from "helpers/math-utils";
 
 const Wrap = ({ contractLoader, accountAddress, vestedId, writeContracts, tx, closeModal }) => {
   // console.log(`writeContracts`, writeContracts);
   const [state, setState] = useState({
-    underlyingAmount: 10,
-    address: accountAddress,
+    underlyingAmount: "",
+    address: "",
```

Comment:
That means i can wrap tokens to another address?

Reply:
Yes, that’s right. Once a VestedERC20 token is deployed anyone can decide to wrap the underlying token (e.g. HNY) to a receiver.